build.sbt

```diff
@@ -164,15 +164,21 @@ commands ++= Seq(
   Command.single("test-mtags-dyn") { (s, scalaV) =>
     crossTestDyn(s, scalaV)
   },
-  Command.args("publish-mtags-dyn", "<scala-version> <version>") {
-    (state, args) =>
-      val configured =
-        configureMtagsScalaVersionDynamically(state, args(0), Some(args(1)))
-      val (out, _) =
-        Project
-          .extract(configured)
-          .runTask(mtags / PgpKeys.publishSigned, configured)
-      out
+  Command.args("publish-mtags-dyn", "<scala-version> <version>") { (state, args) =>
```

Comment:
Can't we just invoke only publishSigned?

In the docs:
```
(optional) CI_RELEASE: the command to publish all artifacts for stable releases. Defaults to +publishSigned if not provided.
```
we could set CI_RELEASE to "mtags/publishSigned"?

Reply:
We also need override `version` + `scalaVersion`

Reply:
We can run `sbt ++<scalaversion> ci-release` maybe? And set the version manually in the project

Reply:
yes we can.
This would require to write in bash smth like:
```sh
if [ $GITHUB_REF == refs/tags*] && [ $GITHUB_REF_NAME == mtags_*]; then
  VERSION=$(echo $GITHUB_REF_NAME | cut -d "_" -f2 )
  SCALA_VERSION=$(echo $GITHUB_REF_NAME | cut -d "_" -f3 )
  # ... validate versions
  CI_RELEASE="mtags/publishSigned"
  sbt "set mtags / version :=\"$VERSION\"; set mtags/scalaVersion := \"$SCALA_VERSION\"; ci-release"
fi
```
But I don't think that it's the best option.